lib/token-assert.js

```diff
@@ -31,19 +31,23 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
         if (nextToken.loc.start.line === token.loc.end.line &&
             (nextToken.loc.start.column - token.loc.end.column) !== spaces
         ) {
+            nextToken.whitespaceBefore = new Array(spaces + 1).join(' ');
```

Comment:
Yeah, I vote for deferring this discussion till later. I believe good assertion layer must cover 100% of cases.